packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx

```diff
@@ -426,7 +439,12 @@ export const CreatePlanFormContainer = ({
 }: {
   isDiscoveryProject: boolean;
 }) => {
+  const { environment } = useContext(ShellContext);
+  const locale = environment.getUserLocale();
+
   const { t } = useTranslation('create');
+  const { addSuccess } = useNotifications();
+  const { t: tListing } = useTranslation('listing');
```

Comment:
```suggestion
  const { t } = useTranslation('listing');
```
You can use namespace if need in the t function

Reply:
```suggestion
  const { t } = useTranslation(['create', 'listing']);
```